.github/workflows/ci-test.yaml

```diff
@@ -70,7 +87,9 @@ jobs:
       - name: Build test image, load images, and run tests
         run: |
           source ./scripts/build.sh
```

Comment:
nit: I'd put `source` next to `load_test_images_into_cluster` -- initially I thought this `source` was unused but turns out it is down there 😄